reference/api/tasks.md

```diff
@@ -292,3 +306,65 @@ Get a single task.
+| :--------------------- | :----------------------------------------------------------------------------------------------------------------------------------- |
+| **`uids`**             | Delete tasks based on `uid`. Separate multiple `uids` with a comma (`,`). Use `uids=*` for all `uids`                                |
+| **`statuses`**         | Delete tasks based on `status`. Separate multiple `statuses` with a comma (`,`). Use `statuses=*` for all `statuses`                 |
+| **`types`**            | Delete tasks based on `type`. Separate multiple `types` with a comma (`,`). Use `types=*` for all `types`                            |
+| **`indexUids`**        | Delete tasks based on `indexUid`. Separate multiple `uids` with a comma (`,`). Use `indexUids=*` for all `indexUids`. Case-sensitive |
+| **`canceledBy`**       | Delete tasks based on the `canceledBy` field                                                                                         |
+| **`beforeEnqueuedAt`** | Delete tasks **before** a specified `enqueuedAt` date                                                                                |
+| **`beforeStartedAt`**  | Delete tasks **before** a specified `startedAt` date                                                                                 |
+| **`beforeFinishedAt`** | Delete tasks **before** a specified `finishedAt` date                                                                                |
+| **`afterEnqueuedAt`**  | Delete tasks **after** a specified `enqueuedAt` date                                                                                 |
+| **`afterStartedAt`**   | Delete tasks **after** a specified `startedAt` date                                                                                  |
+| **`afterFinishedAt`**  | Delete tasks **after** a specified `finishedAt` date                                                                                 |
+
+::: note
+Date filters are exclusive, meaning you can only filter tasks before or after a specified date.
+:::
+
+[To learn more about using the query parameters, refer to our dedicated guide.](/learn/advanced/asynchronous_operations.md#filtering-tasks)
+
+### Example
+
+<CodeSamples id="delete_task_1" />
+
+#### Response: `200 Ok`
+
+```json
+{
+  "taskUid": 3,
+  "indexUid": null,
+  "status": "enqueued",
+  "type": "taskDeletion",
+  "enqueuedAt": "2021-08-12T10:00:00.000000Z"
+}
+```
+
+::: note
+Since `taskDeletion` is a [global task](/learn/advanced/asynchronous_operations.md#global-tasks), its `indexUid` is `null`.
+:::
+
+You can use this `taskUid` to get more details on the [status of the task](#get-one-task).
+
+### Delete all tasks
+
+You can delete all tasks by using the following filter:
+
+<RouteHighlighter method="DELETE" route="/tasks?statuses=failed,canceled,succeeded" />
+
+The API key used must have access to all indexes (`"indexes": [*]`) and the [`task.delete`](/reference/api/keys.md#actions) action.
```

Comment:
This statement is not true. It's possible to delete tasks on indexes you only have access as soon as you have the `task.delete` action.

The statement "The API key used must have access to all indexes" is true when you want to see `taskDeletion` tasks type on `GET /tasks` since it's considered a global task.

@irevoire can you confirm?